src/controller/java/src/chip/devicecontroller/ControllerParams.java

```diff
@@ -126,6 +126,14 @@ public Builder setControllerVendorId(int controllerVendorId) {
       return this;
     }
 
+    /**
+     * Sets the FailsafeTimer duration passed to ChipDeviceCommissioner's 
+     * CommissioningParameters. Increasing this value from its default will allow more time 
+     * for network scans, cloud op cert signing calls, and user interaction.
```

Comment:
This is not quite true. Internal logic to autocommissioner can re-call ArmFailSafe as well to account for network config delays

Reply:
added clarification